Dockerfile.perses-operator

```diff
@@ -0,0 +1,22 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_golang_1.22 as build-env
+
+RUN apk add --update --no-cache make bash mailcap
```

Comment:
This command might not work, or might be necessary anymore because we changed the image. 

If we don't have access to the brew.registry.redhat.io registry -- then try to find an equivalent image. 
Test with docker build <image> to see if it succeeds.